manifests/params.pp

```diff
@@ -20,6 +20,25 @@
   $package_name       = 'vault'
   $package_ensure     = 'installed'
 
+  $checksum_type       = 'sha256'
+  $checksums           = {
```

Comment:
I'd suggest writing a custom `rake` task which fetched all the checksums for a given official vault release.